lbry/testcase.py

```diff
@@ -324,15 +327,18 @@ def on_transaction_address(self, tx, address):
     async def generate(self, blocks):
         """ Ask lbrycrd to generate some blocks and wait until ledger has them. """
         prepare = self.ledger.on_header.where(self.blockchain.is_expected_block)
-        height = self.blockchain.block_expected
+        height = self.blockchain.block_expected + blocks
```

Comment:
I would like to align the styles used in `generate_and_wait()` and `generate()`.

Let me know if the style where `height = self.blockchain.block_expected` is placed **after** `blockchain.generate(n)` is preferred, or whether this style is preferred.

Reply:
Decided to just use the pre-existing style of generate_and_wait().